src/constants.ts

```diff
@@ -1,10 +1,14 @@
 export const TRANSACTION_MIN_GAS_PRICE = 1000000000;
 export const TRANSACTION_OPTIONS_DEFAULT = 0;
 export const TRANSACTION_OPTIONS_TX_HASH_SIGN = 1;
+export const TRANSACTION_OPTIONS_TX_GUARDED = 2;
 export const TRANSACTION_VERSION_DEFAULT = 1;
-export const TRANSACTION_VERSION_TX_HASH_SIGN = 2;
+export const TRANSACTION_VERSION_TX_OPTIONS = 2;
 export const ESDT_TRANSFER_GAS_LIMIT = 500000;
 export const ESDT_TRANSFER_FUNCTION_NAME = "ESDTTransfer";
 export const ESDTNFT_TRANSFER_FUNCTION_NAME = "ESDTNFTTransfer";
 export const MULTI_ESDTNFT_TRANSFER_FUNCTION_NAME = "MultiESDTNFTTransfer";
 export const ESDT_TRANSFER_VALUE = "0";
+
+// Masks
+export const TRANSACTION_OPTIONS_TX_GUARDED_MASK = 0b0010
```

Comment:
Shouldn't we have here an additional mask used for `options for tx signed with hash`? CC: @bogdan-rosianu

Reply:
Yes, I think it's ok if we add this

Reply:
added 
`export const TRANSACTION_OPTIONS_TX_HASH_SIGN_MASK = 0b0001`